packages/react-native/src/primitives/Button/styles.ts

```diff
@@ -1,6 +1,23 @@
-import { StyleSheet } from 'react-native';
+import { Theme } from '../../theme';
 import { ButtonStyles } from './types';
 
-export const styles: ButtonStyles = StyleSheet.create({
+export const baseStyles: ButtonStyles = {
+  borderWidth: 1,
+  borderRadius: 4,
+  padding: 8,
   text: { alignSelf: 'center' },
-});
+};
+
+export const getThemedStyles = (theme: Theme): ButtonStyles => {
+  const buttonToken = theme.tokens?.components.button;
+  return {
```

Comment:
Suggest usage of `StyleSheet.create` here to enforce structure and improve typing:
```
  return StyleSheet.create({
    ...
  })
```